### PR TITLE
Azure storage mount for backup on ES

### PR DESCRIPTION
**Pull request by stefansaftic**
Support for azure storage mount added for backup purpose

### CODE REVIEW

File: deploy_elasticsearch.json [hunk 1]
```diff
@@ -42,6 +42,15 @@
         "userScriptBase": {
             "type": "String"
         },
+        "backupStorageAccountName": {
+            "type": "String"
+        },
+        "backupStorageAccountKey": {
+            "type": "String"
+        },
+        "basckupShareName": {
```
Comment (denic) [inline deploy_elasticsearch.json]: typo?

File: elasticsearch-install.sh [hunk 2]
```diff
@@ -7,19 +7,30 @@ scriptBase=$(echo $5 | grep -P -o ".*/")
...
 echo "datacenter=$DATACENTER"
 echo "masterVmName=$MASTERVMNAME"
 echo "adminUserName=$ADMINUSER"
 echo "nodeType=$NODETYPE"
 echo "scriptBase=$scriptBase"
 echo "targetEnv=$targetEnv"
 echo "userScriptBase=$userScriptBase"
+echo "backupStorageAccountName=$backupStorageAccountName"
+echo "backupStorageAccountKey=$backupStorageAccountKey"
+echo "basckupShareName=$basckupShareName"
```
Comment (denic) [inline elasticsearch-install.sh]: typo

File: elasticsearch-install.sh [hunk 3]
```diff
@@ -7,19 +7,30 @@ scriptBase=$(echo $5 | grep -P -o ".*/")
...
 
 if [[ -z "$userScriptBase" ]] ; then
   echo "userScriptBase missing"
   exit 1
 fi
 
+if [[ -z "$basckupShareName" ]] ; then
+  echo "basckupShareName missing"
```
Comment (denic) [inline elasticsearch-install.sh]: typo

File: deploy_elasticsearch.json (hunk 1 above)
Comment (stefansaftic) [reply deploy_elasticsearch.json]: fixed

**Comment by denic** [comment]
This is on hold.

**Pull request closed without merging**